tests/setup_suite.bash

```diff
@@ -0,0 +1,6 @@
+setup_suite() {
+    # bats::on_failure hook requires >= 1.12.0
+    bats_require_minimum_version 1.11.0
```

Comment:
Is this correct? should it be 1.12.0?

Reply:
It does not really require 1.12.0, it will still work fine with 1.11.0 (just on_failure triggers won't work).
I can't use `bats_require_minimum_version 1.12.0` because we use ubuntu runners that don't have new version yet.